tests/data/bundles/test_yahoo.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import division
```

Comment:
Already merged with origin/master, so a few of these updates are from the recent PR's